pydfs_lineup_optimizer/sites/draftkings/classic/importer.py

```diff
@@ -77,6 +87,8 @@ def import_lineups(self, players):
                 for index, position in zip(range(start_column, end_column), position_names):
                     try:
                         player_data = line[index]
+                        if player_data.find('(LOCKED)') >= 0: # Remove possible '(LOCKED)' substring
```

Comment:
You can simply make `player_data = player_data.replace('(LOCKED)', '')` for removing locked text